dkeycache/Enclave/enclave_ra.cpp

```diff
@@ -41,8 +41,6 @@
 #include "sgx_tkey_exchange.h"
 #include "sgx_tcrypto.h"
 
-#define SGX_AES_KEY_SIZE 16
-
 #define SGX_DOMAIN_KEY_SIZE     16
```

Comment:
this also defined in the datatypes.h, so please remove it.

Reply:
copy